app.py

```diff
@@ -119,12 +119,33 @@ def auth_granted():
     @app.route('/api/slack/commands', methods=['POST'])
     def slash_command():
         data = request.form.to_dict()
+        print('data: ', data)
```

Comment:
now's a good time to think about debugging

let's make this conditional on app.debug

```
if app.debug:
    print('slash_command data: ', data)
```